interactive_engine/sdk/backup_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -x
+base_dir=$(cd "$(dirname "$0")"; pwd)
+ps -ef | grep "com.alibaba.maxgraph.servers.MaxNode" | grep -v grep | awk '{print $2}' | xargs kill -9
+cd ${base_dir}/.. && mvn clean install -DskipTests -Pv2
+cd ./distribution/target/ && tar xvzf maxgraph.tar.gz && cd maxgraph
```

Comment:
I suggest split the `install` stage and `test` stage into two steps in local.yml.

Reply:
done